src/NewsService.Data/NewsRepository.cs

```diff
@@ -16,10 +15,14 @@ namespace LT.DigitalOffice.NewsService.Data
     public class NewsRepository : INewsRepository
     {
         private readonly IDataProvider _provider;
+        private readonly IHttpContextAccessor _httpContextAccessor;
 
-        public NewsRepository(IDataProvider provider)
+        public NewsRepository(
+            IDataProvider provider,
+            IHttpContextAccessor httpContextAccessor)
         {
             _provider = provider;
+            _httpContextAccessor = httpContextAccessor;
         }
 
         public bool EditNews(Guid newsId, JsonPatchDocument<DbNews> news)
```

Comment:
please rename news to request or smth else

Reply:
i talk about name of 2d argument

Reply:
done